.github/workflows/nm-upload-assets-to-gcp.yml

```diff
@@ -31,6 +35,14 @@ on:
                 description: 'git commit hash or branch name'
                 type: string
                 required: true
+            wf_category:
+                description: "workflow category: REMOTE, NIGHTLY, RELEASE"
+                type: string
+                default: "REMOTE"
+            whl:
```

Comment:
We don't need this whl argument anymore as we're getting the name from the .github/actions/nm_whl_tar_gz_names/action.yml?

Also can we rename the name of this workflow to nm-upload-assets.yml since we do upload to both public pypi and gcp now?

Reply:
go catch.  I got the one in `workflow_call`, but missed the one in `workflow_dispatch`.  Updated and changed the workflow name too.